std/src/main/scala/algebra/std/boolean.scala

```diff
@@ -9,6 +10,13 @@ package object boolean extends BooleanInstances
 trait BooleanInstances {
   implicit val booleanAlgebra: BooleanAlgebra =
     new BooleanAlgebra
+
+  val BooleanRing = new BoolRing[Boolean] {
```

Comment:
@johnynek I can't find an instance of `Field[Boolean]` in `algebra`. Am I just not looking in the right places?